### PR TITLE
Implement Ollehmusic Connector

### PR DESCRIPTION
**Pull request by KwonDae**
Please check my request about implement ollehmusic connector :)

### CODE REVIEW

File: src/connectors/olleh.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+ Connector.playerSelector = '#n11_player';
```
Comment (inverse) [inline src/connectors/olleh.js]: Please remove leading spaces in this file.

**Comment by KwonDae** [comment]
I remove space that file. please check the changes.

**Comment by inverse** [comment]
Was more that spacing before - e.g. there was a leading ` ` on each line.

Could you resolve the merge conflict and then we should be good :+1:

**Comment by KwonDae** [comment]
Sorry, I remove the space of each line. please check the file :)

**Pull request closed without merging**